src/utils/survey-utils-test.js

```diff
@@ -217,7 +217,20 @@ describes.realWin('Survey utils', (env) => {
           eventType: AnalyticsEvent.EVENT_SURVEY_DATA_TRANSFER_COMPLETE,
           eventOriginator: EventOriginator.SWG_CLIENT,
           isFromUserAction: true,
-          additionalParameters: null,
+          additionalParameters: new EventParams([
```

Comment:
Could we do this once at the top in a const?  const EVENT_PARAMS = ...

Reply:
Done